AUTHORS

```diff
@@ -0,0 +1,2 @@
+Stanley Osagie
+Felex Masereka
```

Comment:
good job so far
correct work flow
linters failing
please do well to correct your linters







g



great